Content.Client/Verbs/UI/ConfirmationMenuElement.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Content.Client.ContextMenu.UI;
+using Content.Shared.Verbs;
+using Robust.Shared.Maths;
+using Robust.Shared.Utility;
+
+namespace Content.Client.Verbs.UI;
+
+public partial class ConfirmationMenuElement : ContextMenuElement
+{
+    public const string StyleClassConfirmationContextMenuButton = "confirmationContextMenuButton";
+
+    public readonly Verb Verb;
+    public readonly VerbType Type;
+
+    public override string Text
+    {
+        set
+        {
+            var message = new FormattedMessage();
+            message.PushColor(Color.White);
+            message.AddMarkupPermissive(value.Trim());
+            Label.SetMessage(message);
+        }
+    }
+
+    public ConfirmationMenuElement(Verb verb, string? text, VerbType type) : base(text)
+    {
+        Verb = verb;
+        Type = type;
+
```

Comment:
```suggestion
        Type = type;
        Icon.Visible = false;

```
Or slap a question mark or warning/exclamation mark in there or something. Currently its just an empty space.

Reply:
Though obviously if you decide to replace the expansion-indicator icon with a symbol, then this should just be invisible and vice versa